core/client.go

```diff
@@ -18,6 +18,8 @@ type Client = client.Client
 func NewRemote(protocol, remoteAddr string) (Client, error) {
 	httpClient := retryhttp.NewClient()
 	httpClient.RetryMax = 2
+	// suppress logging
+	httpClient.Logger = nil
```

Comment:
I'd also prefer this to use the main logger used in this package instead of nil but this is still better than before.